functions/imagemagick/main.py

```diff
@@ -81,6 +81,19 @@ def __blur_image(current_blob):
     blur_bucket_name = os.getenv("BLURRED_BUCKET_NAME")
     blur_bucket = storage_client.bucket(blur_bucket_name)
     new_blob = blur_bucket.blob(file_name)
+
+    # Copy mimetype from the source blob.
+    new_blob.content_type = current_blob.content_type
+
+    # Add custom metadata.
+# Add custom metadata.
+    import datetime
+    new_blob.metadata = {
+        "blurred": "true",
+        "original_file": file_name,
+        "blurred_at": datetime.datetime.utcnow().isoformat(),
+    }
+
```

Comment:
Is there a reason why you're adding this functionality? 

For context: This is one of multiple implementations of the same functionality across multiple programming languages. We try and keep all implementations functionality the same, so adding this needs to be considered in the context of other samples.

Reply:
This sample is designed to demonstrate [Cloud Run functions, the Cloud Vision API, and ImageMagick to detect and blur offensive images that get uploaded to a Cloud Storage bucket](https://cloud.google.com/functions/1stgendocs/tutorials/imagemagick-1st-gen). It is not designed for your classroom. You are free to reuse the code for your own requirements as long as you adhere to the open source license attached.